### PR TITLE
Upload files

### PR DESCRIPTION
**Pull request by kifion**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+import 'localization/locale_constant.dart';
+import 'localization/localizations_delegate.dart';
+
+void main() {
+  runApp(GuessNumberApp());
+}
+
+class GuessNumberApp extends StatefulWidget {
```
Comment (vasilevkin) [inline lib/main.dart:12]: move to separate file app.dart

File: lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    super.didChangeDependencies();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      builder: (context, child) {
+        return MediaQuery(
+          child: child,
+          data: MediaQuery.of(context).copyWith(textScaleFactor: 1.0),
```
Comment (vasilevkin) [inline lib/main.dart:49]: use constant instead of magic numbers

File: lib/main.dart [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        );
+      },
+      title: 'Guess the number',
```
Comment (vasilevkin) [inline lib/main.dart:52]: should be localizable

File: lib/screens/home.dart [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+                  padding: const EdgeInsets.all(paddingSize),
+                  child: Text(Languages.of(context).result + ": " + result,
+                      style: TextStyle(fontSize: fontSize))),
+            ],
+          ),
+        ),
+      );
+}
+
+class IconSettingsWidget extends StatelessWidget {
```
Comment (vasilevkin) [inline lib/screens/home.dart:105]: move to separate file

**Review by vasilevkin — CHANGES_REQUESTED** [review]
request changes

File: lib/localization/locale_constant.dart [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import 'dart:ui';
```
Comment (vasilevkin) [inline lib/localization/locale_constant.dart:1]: ══╡ EXCEPTION CAUGHT BY GESTURE ╞═══════════════════════════════════════════════════════════════════
The following NoSuchMethodError was thrown while handling a gesture:
The method '_greaterThanFromInteger' was called on null.
Receiver: null
Tried calling: _greaterThanFromInteger(1)

When the exception was thrown, this was the stack:
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
#1      int.> (dart:core-patch/integers.dart:105:18)
#2      HomeState._onButtonPressed (package:RandomNumber/screens/home.dart:32:28)
#3      HomeState.build.<anonymous closure> (package:RandomNumber/screens/home.dart:82:42)
#4      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:992:19)
#5      _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:1098:38)
#6      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:184:24)
#7      TapGestureRecognizer.handleTapUp (package:flutter/src/gestures/tap.dart:524:11)
#8      BaseTapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:284:5)
#9      BaseTapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:219:7)
#10     PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:477:9)
#11     PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:78:12)
#12     PointerRouter._dispatchEventToRoutes.<anonymous closure> (package:flutter/src/gestures/pointer_router.dart:124:9)
#13     _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:377:8)
#14     PointerRouter._dispatchEventToRoutes (package:flutter/src/gestures/pointer_router.dart:122:18)
#15     PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:108:7)
#16     GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:220:19)
#17     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:200:22)
#18     GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:158:7)
#19     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:104:7)
#20     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:88:7)
#24     _invoke1 (dart:ui/hooks.dart:267:10)
#25     _dispatchPointerDataPacket (dart:ui/hooks.dart:176:5)
(elided 3 frames from dart:async)

Handler: "onTap"
Recognizer:
  TapGestureRecognizer#38267
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by gesture ═══════════════════════════════════════════════════════════════
The method '_greaterThanFromInteger' was called on null.
Receiver: null
Tried calling: _greaterThanFromInteger(1)
════════════════════════════════════════════════════════════════════════════════════════════════════

Another exception was thrown: NoSuchMethodError: The method '_greaterThanFromInteger' was called on null.

File: lib/localization/locale_constant.dart [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+import 'package:flutter/material.dart';
```
Comment (vasilevkin) [inline lib/localization/locale_constant.dart:3]: check localization when the app is installed from scratch

**Comment by vasilevkin** [comment]
use snake_case_notation for package/files/project names

File: lib/localization/locale_constant.dart (hunk 5 above)
Comment (kifion) [reply lib/localization/locale_constant.dart]: import was deleted

File: lib/localization/locale_constant.dart (hunk 6 above)
Comment (kifion) [reply lib/localization/locale_constant.dart:1]: removed dar ui.

**Pull request merged**